src/viem/getLockableWallet.test.ts

```diff
@@ -29,6 +29,9 @@ jest.mock('src/viem', () => {
   }
 })
 
+const PRIVATE_KEY1 = '0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef'
+const ACCOUNT_ADDRESS1 = privateKeyToAddress(PRIVATE_KEY1).toLowerCase() as Address
```

Comment:
We now use these in several tests, I'll move then to `test/values` in a follow up PR.

Reply:
Done in https://github.com/valora-inc/hooks/pull/501